python/src/opendp/extras/polars/__init__.py

```diff
@@ -112,7 +112,7 @@ def noise(
         :example:
 
         >>> import polars as pl
-        >>> dp.len()
+        >>> print(dp.len())
         len()...:noise([...])
```

Comment:
Is the print necessary now?

Going back to a comment from the first round, while this may be useful as a test, I'm not sure how useful it is for a reader of the docs -- I don't think it's an "example" of something the user would typically do with the method.